src/components/charts/helper.test.js

```diff
@@ -127,4 +146,8 @@ describe('helper', () => {
     expect(charts.summaries).toEqual([summaryData]);
     expect(charts.stackedBarCharts).toEqual([]);
   });
+
+  it('return selecetd values from SQON as expected', () => {
+    expect(helper.getSQONValues(selectWhiteSqon, 'ethnicity')).toEqual(selectWhiteSqonValues);
+  });
```

Comment:
I think we need some more tests. what if there is a filter with no values selected? does it return the length of the buckets or the sqon count? things like that.